polaris-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -11,11 +11,14 @@ import styles from './Breadcrumbs.scss';
 
 export interface BreadcrumbsProps {
   /** Collection of breadcrumbs */
-  breadcrumbs: (CallbackAction | LinkAction)[];
+  breadcrumbs: (CallbackAction | LinkAction) | (CallbackAction | LinkAction)[];
 }
 
 export function Breadcrumbs({breadcrumbs}: BreadcrumbsProps) {
-  const breadcrumb = breadcrumbs[breadcrumbs.length - 1];
+  const breadcrumb =
+    breadcrumbs instanceof Array
```

Comment:
```suggestion
    Array.isArray(breadcrumbs)
```

There's a few instances of this that can be updated 👀

Reply:
Totally. Didn't know what was preferred